api/src/main/java/org/datacleaner/configuration/RemoteServerState.java

```diff
@@ -22,6 +22,55 @@
 /**
  * States for connection to remote components.
  */
-public enum RemoteServerState {
-    UNKNOWN, OK, ERROR;
+public class RemoteServerState {
+
+    public enum State {
+        UNKNOWN, OK, ERROR, NO_CREDIT;
```

Comment:
UNKNOWN is base state before real check or when user isn't connect to DataCloud.
NOT_CONNECTED sounds like error state for me.